src/api/app/controllers/person/token_controller.rb

```diff
@@ -2,6 +2,7 @@
 module Person
   class TokenController < ApplicationController
     before_action :set_user
+    before_action :set_package, only: [:create]
```

Comment:
Adding this `before_action` here would call `set_package` even the user is not authorized.

Before the changes of this pull request, checking for the permissions of the user was done before searching for the package. Could we keep this behaviour?